topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.md

```diff
@@ -0,0 +1,352 @@
+indicator of probability of taxon presence for each environmental layer pixel. Here, the method boosted regression trees (BRT) was used to adjust the relationship between the presence of a single 
+taxon and the environmental conditions under which the taxon has been detected. BRT modelling is based on a learning algorithm automatic using iterative classification trees. 
+
+**What it does ?**
+
+Two treatments are performed in this tool: the creation of the taxa distribution model and the use of this model to obtain a prediction index. The prediction index obtained from each BRT model for each pixel of the environmental layers is an approximation of the probability of detection of the presence of the taxon.
+
+This tool gives in output a file containing the predictions of the probability of the presence of each taxon for each pixel (latitude, longitude) environmental, a visualization of these pixels for each taxon and graphs showing the percentage of model explanation for each environmental parameter. We'r gonna go back to this in the following workflow you have an example of these files below. 
+
+**How to use it ?**
+
+> <hands-on-title> Run the BRT tool </hands-on-title>
+>
+> 1. {% tool [BRT tool prediction](toolshed.g2.bx.psu.edu/repos/ecology/ecoregion_brt_analysis/ecoregion_brt_analysis/0.1.0+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Environment Data"*: `env.tsv` (Input dataset containing your environement data)
+>    - {% icon param-file %} *"Occurrences Data File"*: `occurrences.tsv` (Input dataset(s) containing your species occurrences. You can select multiple datasets)
+>
+>      {% snippet faqs/galaxy/tools_select_multiple_datasets.md %}
+>
+>    - *"Write the name of your abiotic parameters (comma separated)"*: `Carbo,Grav,Maxbearing` (This is an example. You need to write the exact name of your environement parameters present in your environement file and your occurrences file(s))
+>
+> Remember : Your abiotic parameters must be present in your occurrence data file(s) and must be named the same as in your environment file. 
+>    
+> 2. Check your outputs. You must have four outputs collections.
+>
+>    - Prediction files
+>    - Validation files (Taxa, AUC, Tree complexity, Total deviance explained)
+>    - Species distribution prediction maps
+>    - Partial dependence plots
+>
+{: .hands_on}
+
+In the 'Prediction files' collection there must be a file containing predictions of the probability of the presence of each taxon for each environmental pixel (latitude, longitude) for each
+the occurrence files you entered.
+
+In the collection 'Validation files' there must be a file containing for each taxon the validation metrics of the associated model.
+
+In the collection 'Species distribution prediction map' there must be for each taxon a map representing their probability of presence at each environmental layer pixel. 
+Here is an example:
+
+![](./Images/BRT-Thouarella_vulpicauda_pred_plot.png)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*

Reply:
Thank you Helena! I tried someting for both alt-text!